src/N7consulting/PrivacyBundle/Resources/views/Privacy/index.html.twig

```diff
@@ -51,8 +51,11 @@
     {{ parent() }}
     <script type="text/javascript">
         $('.select2').select2();
-
-        /** Create Confirm message */
+        /** Hide search on second dropdown **/
```

Comment:
The "serach with text" was useless in this field, but I want to keep the select2 style.